tests/unit/amo/pages/TestUsersUnsubscribe.js

```diff
@@ -33,19 +30,12 @@ describe(__filename, () => {
   };
 
   const render = ({
-    i18n = fakeI18n(),
     params = getParams(),
     store = dispatchClientMetadata().store,
     ...props
   } = {}) => {
-    return shallowUntilTarget(
-      <UsersUnsubscribe
-        i18n={i18n}
-        match={{ params }}
-        store={store}
-        {...props}
-      />,
-      UsersUnsubscribeBase,
+    return defaultRender(
```

Comment:
This is a page, so we should be using `renderPage` from `tests/helpers` as opposed to `render`. This will mean we cannot pass in any `props`, but that should be doable. Also, just for future reference, we shouldn't be passing `store` in as a property to anything we are rendering, instead, we pass `store` into the `render` method as an option: `render(<MyComponent />, {store})`.

Reply:
You're going to have to use `params` to build a `location` in this scenario.